src/test/java/com/redislabs/redisai/ModelTest.java

```diff
@@ -0,0 +1,74 @@
+package com.redislabs.redisai;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ModelTest {
+
+    @Test
+    public void getSetTag() {
+        Model model = new Model(Backend.ONNX, Device.GPU, new String[0], new String[0], new byte[0]);
+        String tag = model.getTag();
+        Assert.assertEquals(tag, "");
```

Comment:
@filipecosta90 I think there should be a different between empty string and null (tag wasn't set)

Reply:
Got it. changing :+1: